modules/basics/src/main/resources/com/opengamma/basics/date/BusinessDayConvention.ini

```diff
@@ -0,0 +1,41 @@
+# ExtendedEnum Business Day configuration
+
+# Additional files may be added to the classpath with the same name
+# All files will be loaded, and the priority and chain fields used to determine how they are processed
+# The 'priority' value is used to select which file to process first, higher numbers have higher priority
+# The 'nextInChain' value is used to determine whether to chain on to the next lower priority file
+# The 'removedSections' value is a list of sections to remove from lower priority chained files
```

Comment:
On second thoughts, do we definitely need ``removedSections``? Would it be simpler to understand if we just said you can override values from lower priority files but not remove them? And if you need some but not all of that file's values, just copy them into your file and don't chain.